cmd/query/app/flags.go

```diff
@@ -71,22 +74,10 @@ type QueryOptionsBase struct {
 	MaxClockSkewAdjust time.Duration `mapstructure:"max_clock_skew_adjust"  valid:"optional"`
 	// EnableTracing determines whether traces will be emitted by jaeger-query.
 	EnableTracing bool `mapstructure:"enable_tracing"`
-}
-
-// QueryOptions holds configuration for query service
-type QueryOptions struct {
-	QueryOptionsBase
-
-	// AdditionalHeaders
-	AdditionalHeaders http.Header
```

Comment:
just to confirm, AdditionalHeaders only applied to HTTP protocol? Does OTEL have similar construct for gRPC?